src/main.cpp

```diff
@@ -33,6 +49,10 @@ int main()
 
         window.clear();
         window.draw(sprite);
+        if (font.loadFromFile("resources/OpenSans-Regular.ttf"))
+        {
+            window.draw(text);
+        }
```

Comment:
I don't think this will even compile. The `text` object won't be available here.

Reply:
Yup, you can see that the automated build failed with two error:
https://github.com/ufrshubham/sfml-project-template/actions/runs/7609957296/job/20723732956#step:6:92
https://github.com/ufrshubham/sfml-project-template/actions/runs/7609957296/job/20723732956#step:6:96